tests/topotests/cspf_topo1/r1/zebra.conf

```diff
@@ -10,6 +10,7 @@ interface r1-eth0
  link-params
   metric 20
   delay 10000
+  max-bw 10e+10
```

Comment:
Please, be consistent with ospf and isis te topo test for the value choose between 10e+10 or 1e+10

Reply:
done